RockPaperScissors/RockPaperScissors/UserCardsError.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  UserCardsError.swift
+//  RockPaperScissors
+//
+//  Created by 박종화 on 2023/05/03.
+//
+
+enum UserCardsError: Error {
+    case firstGameInputError
+    case SecondGameInputError
```

Comment:
Error 프로토콜을 채택하여 오류를 정의해주신 부분 좋습니다 👍👍👍

다만, firstGame이든 secondGame이든 InputError로 같다는 점에서 네이밍에 개선할 점이 보이네요!
그리고 카멜 케이스 적용에 유의해주세요 🙂
```suggestion
enum InputError: Error {
    case invalidInput
```

Reply:
코드를 수정해가면서 해당 부분을 놓친 것 같습니다. 현재는 따로 구분할 필요가 없어져서 하나의 case로 수정했습니다!!!